content/developers/index.md

```diff
@@ -1,7 +1,148 @@
 ---
 title: Quick start
-description: Our mission as Mysterium Node runners
+description: How to setup your environment
 ---
 
 ## Quick start for developers
-...
+
+Setting up your environment for building, running and testing Mysterium node doesnt have to be hard.
+The minimum required software to set up is:
+
+| Software           | Minimum Version |
+|--------------------|-----------------|
+| Go                 | 1.14.0+         |
+| Docker             | 19.0+           |
+| Docker Compose     | 1.21.0+         |
+| Mage (Recommended) | 1.10+           |
+
+
+<details><summary>General (Mage)</summary>
+<p>
+
+[Mage](https://github.com/magefile/mage) is a simple make-like Go tool we use to help us develop our software.
+Before installing Mage, make sure you've completed the below steps for system setup
```

Comment:
Step order matters, if this step requires having go installed, go installation should be a step before this one.